api_testing/apis/roles_api.py

```diff
@@ -85,3 +85,9 @@ def get_random_role(self):
         roles = response['roles']
         editable_roles = [role for role in roles if role['name'] not in ['Admin', 'Contact']]
         return editable_roles
+
+    def get_role_id_from_name(self, name):
+        response, payload = RolesAPI().get_all_roles()
```

Comment:
it should be 
`        response, payload = self.get_all_roles()
`

Reply:
done